porcupine/utils.py

```diff
@@ -176,6 +175,8 @@ def show(self) -> None:
             # the label will have light text on a light background or
             # dark text on a dark background on some systems.
             tkinter.Label(tipwindow, text=self.text, border=3, fg="black", bg="white").pack()
+            tipwindow.update()
```

Comment:
```suggestion
            tipwindow.update_idletasks()
```

I'm being a little nit-picky here, but usually it's better to use `update_idletasks` instead of `update`, unless you have a good reason not to.

Reply:
Thanks, just updated!